frontend/src/admin/__tests__/CreateUserPageContents.test.jsx

```diff
@@ -45,46 +35,33 @@ describe("<CreateUserPageContents/> class component", () => {
 
   describe("render() returns a component that", () => {
     it("contains one <CreateUserFormConsole/> component with handleSubmit passed as prop", () => {
```

Comment:
Test name no longer accurate (there is no CreateUserFormConsole in here anymore)

Reply:
06019e5